GTViewController/Classes/Nav Classes/GTViewController.m

```diff
@@ -1015,6 +1022,8 @@ - (void)hideNavToolbar {
         
     }
     
+    self.navToolbarIsShown		= NO;
```

Comment:
It seemed like there was a race condition going on with button taps when I was debugging.  I could see justification for this going back, as I think I solved the problem another way.